Las/Utils/Authorization.cs

```diff
@@ -0,0 +1,194 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Text;
+
+using Lucidtech.Las.Core;
+
+namespace Lucidtech.Las.Utils
+{
+    public class AmazonAuthorization
+    {
+        /* General information */
+        private string Region { get; }
+        private string Service { get; }
+
+        /* Keys the customer receives from Lucidtech */
+        private string AwsApiKey { get; }
+        private string AwsAccessKey { get; }
+        private string AwsSecretKey { get; }
+        
+        private const string Algorithm = "AWS4-HMAC-SHA256";
+
+        public AmazonAuthorization(Credentials credentials)
+        {
+            AwsApiKey = credentials.ApiKey ;
+            AwsSecretKey = credentials.SecretAccessKey;
+            AwsAccessKey = credentials.AccessKeyId;
+            Region = "eu-west-1";
+            Service = "execute-api";
+        }
+
+        public Dictionary<string, string> SignHeaders(Uri uri, string method, byte [] body)
```

Comment:
Should whitespace between "byte" and "[]" be removed? e.g. "byte[]" instead of "byte []" ?

Reply:
yes